src/cookbook/use-multi-signature-transactions.md

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Use Multi-Signature Transactions | Cookbook'
+head:
+  - - meta
+    - name: description
+      content: 'Learn how to use multi-signature transactions in Iroha.'
+  - - meta
+    - name: keywords
+      content: 'multi-signature transactions'
+---
+
+# How to Use Multi-Signature Transactions
+
+We will create an account with a key and set a rule that all transactions
+made by this account should be also signed by an additional key. Then we
+will perform a transaction itself, in two stages: first will be signed and
+submitted with the main account key, and then the same transaction will be
+submitted with an additional signature by the second key. To check if
+everything works, we will also create and asset and the multi-signature
+transaction will mint that asset.
+
+So, our first transaction will:
+
+- Create account `mad_hatter` in `wonderland` domain with `key1`
+- Assign `SignatureCheckCondition` to it, which will enforce transactions
+  from `mad_hatter@wonderland` to have an additional `key2`
+- Register a numeric asset `casomile#wonderland` of type `Quantity` and
+  infinite mintability.
```

Comment:
```suggestion
## Example

1. Let's create an account that would need to have an additional signature for its transactions. Let's also create an asset that this account will later mint in order to check that the rule about second signature works as expected.

   The code below covers the following steps:

   - Create account `mad_hatter` in `wonderland` domain with `key1`.
   - Assign `SignatureCheckCondition` to the created account. This will enforce transactions from `mad_hatter@wonderland` to have an additional `key2`.
   - Register a numeric asset `casomile#wonderland` of type `Quantity` and infinite mintability.
```